src/main/resources/default/taglib/w/codelistMultiSelect.html.pasta

```diff
@@ -21,7 +21,7 @@
            items="@Injector.context().getPart(CodeLists.class).getEntries(list)">
         <option value="@entry.getCodeListEntryData().getCode()"
                 @selected="value.contains(entry.getCodeListEntryData().getCode())">
-            @entry.getCodeListEntryData().getValue()
+            @entry.getCodeListEntryData().getTranslatedValue(NLS.getCurrentLang())
```

Comment:
the same probably also applis to code-list-entries.html.pasta (?)

Reply:
if so, it would rather be in multiLanguageTextField.html.pasta. Will check, if we need it there

Reply:
My bad, was thinking about the code-list-entry.html.pasta. But I think in both cases (when viewing the list of entries for a codelist, or editing an individual entry) it makes sense to display the clear text.

We still needed it for the job param template though